test/boulder-config.json

```diff
@@ -2,7 +2,8 @@
   "syslog": {
     "network": "",
     "server": "",
-    "stdoutlevel": 6
+    "stdoutlevel": 6,
+    "sysloglevel": 4
```

Comment:
I believe it's favoured to introduce the new parameter only in `boulder-config-next.json` and to let the zero value be used for `boulder-config.json`. Could you remove it from this config?